signer/threshold_signer.go

```diff
@@ -24,6 +24,8 @@ type ThresholdSigner interface {
 	Sign(req CosignerSignRequest, m *LastSignStateWrapper) (CosignerSignResponse, error)
 
 	GetID() (int, error)
+
+	Stop()
```

Comment:
Worth documenting the expectations around `Stop()` here. It looks like in this case, it is a blocking call that stops any pending goroutines, as opposed to sending an async "stop signal" and requiring a separate `Wait()` call to block on outstanding work.

Reply:
comment added